ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-internet-exit.j2

```diff
@@ -0,0 +1,37 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - router internet-exit #}
+{% if router_internet_exit is arista.avd.defined %}
+!
+router internet-exit
+{%     if router_internet_exit.exit_groups is arista.avd.defined %}
+{%         for exit_group in router_internet_exit.exit_groups | arista.avd.default([]) %}
```

Comment:
you dont need both lines, the default will take care on the missing variable
```suggestion
{%     for exit_group in router_internet_exit.exit_groups | arista.avd.default([]) %}
```

Reply:
Done!